### PR TITLE
Remove More In This Section button from VAMC pages

### PR DESCRIPTION
**Pull request by mpelzsherman**
https://github.com/department-of-veterans-affairs/va.gov-team/issues/10002

## Description

Removes the **More In This Section** button from VAMC pages

This was done in a previous PR, #13157, but it was done too broadly.

This PR should only affect VAMC location pages.

## Testing done
Manual

## Screenshots
VA_Pittsburgh_Health_Care___Veterans_Affairs

## Acceptance criteria
- [x] The **More In This Section** button does not appear on VAMC location pages, but DOES still appear on other pages, e.g. /health-care/eligibility.
- [x] The **More In This Section** button should not appear anywhere the "In This Section" hamburger menu appears.

## Definition of done
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: src/site/components/navigation-sidebar-trigger.html [hunk 1]
```diff
@@ -8,39 +8,37 @@
 used in a template with navigation-sidebar.html
 
 {% endcomment %}
-{% if buildtype == 'vagovprod' %}
```
Comment (mpelzsherman) [inline src/site/components/navigation-sidebar-trigger.html:11]: I removed this conditional so the button would appear in all environments.

**Review by ncksllvn — APPROVED** [review]

**Comment by brangi** [comment]
@mpelzsherman, were you planning to add a smoke cypress to verify the correct UI ?

**Comment by mpelzsherman** [comment]
Per @ncksllvn in [Slack](https://dsva.slack.com/archives/C52CL1PKQ/p1599590588364900?thread_ts=1599589601.362100&cid=C52CL1PKQ)

> The only layouts I know of that should keep the "more in this section" button are the page.drupal.liquid and landing_page.drupal.liquid.

**Comment by mpelzsherman** [comment]
> @mpelzsherman, were you planning to add a smoke cypress to verify the correct UI ?

Yes - I added one to `src/site/tests/vamc/smoke.cypress.spec.js`

File: src/site/tests/vamc/smoke.cypress.spec.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    cy.get('#sidebar-nav-trigger').should('not.exist');
+    cy.get('#sidenav-menu').should('exist');
+    cy.get('h1').contains('VA Pittsburgh health care');
+    cy.get('h2').contains('Locations');
+    cy.get('h3').contains('Manage your health online');
+    cy.get('#stories').contains('Stories');
+    cy.get('#events').contains('Events');
+  });
+});
```
Comment (brangi) [inline src/site/tests/vamc/smoke.cypress.spec.js:18]: Can we add a test that verifies the `side_bar` on mobile layout

Screen Shot 2020-09-09 at 6 43 53 PM

Comment (mpelzsherman) [reply src/site/tests/vamc/smoke.cypress.spec.js:18]: Please explain. I added an ID to that element and my test is checking that it exists:

```js
cy.get('#sidenav-menu').should('exist');
```

What other verification did you have in mind?

Comment (brangi) [reply src/site/tests/vamc/smoke.cypress.spec.js:18]: I actually did not see the viewPort called `cy.viewport(481, 1000);` now makes sense

**Review by brangi — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]

**Pull request merged**